.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
```

Comment:
Add a blank line here too

Reply:
> Then we may need to add the versions supported in the documentation itself.

In Readme the mentioned version is python 3.7+ that's why i have used python 3.7 @devkapilbansal 
https://github.com/anitab-org/open-source-programs-backend#tech-stack